### ISSUE TITLE
Delayed presentation of UISplitViewController's content breaks on iPhone with the iOS 13 beta sdk

### ISSUE THREAD

**Issue by nataliq** [iZettle/Presentation#40]
# Expected Behavior 
 
Running the Messages example app on iPhone running iOS 13 should show the list of messages.
 
# Current Behavior 

An empty grey screen is shown.
 
# Steps to Reproduce 
 
1. Build and run the Messages example app with Xcode 11 on an iPhone simulator.
 
## Context  
 
* Operating Version: iOS 13 beta 1
* Swift version: 5.1

In the main `present` method of `UIViewController` [we delay the presentation](https://github.com/iZettle/Presentation/blob/master/Presentation/UIViewController+Presentation.swift#L22) if the view controller is not loaded yet. It seems like setting the `viewControllers` of a `UISplitViewController` after it was presented in automatic mode on iPhone (showing only one of the screens), doesn't add the view controllers to the screen even though they are assigned to the property. This can be reproduced by replacing[ this line of code](https://github.com/iZettle/Presentation/blob/master/Examples/Messages/Example/Messages.swift#L47) of the example app: 
```swift
bag += split.present(viewController, options: [ .defaults, .showInMaster ])
``` 
with 
```swift
let nc = customNavigationController([ .defaults, .showInMaster ])
nc.viewControllers = [viewController]

DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(2)) {
    split.viewControllers = [nc]
}
```

Removing the `asyncAfter` call results in a correct behaviour.
 
## Failure Logs 
 
No logs.

**Comment by nataliq** [comment]
Dirty fix for the time being: 

Add `self is UISplitViewController ||` to the [guard statement](https://github.com/iZettle/Presentation/blob/master/Presentation/UIViewController+Presentation.swift#L22) in `UIViewController+Presentation`.

**Comment by niil-ohlin** [comment]
Just to make sure that I understood it correctly. 


```swift
// 1
bag += split.present(viewController, options: [ .defaults, .showInMaster ])
```

and

```swift
// 2
let nc = customNavigationController([ .defaults, .showInMaster ])
nc.viewControllers = [viewController]

DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(2)) {
    split.viewControllers = [nc]
}
```

does not work as expected. But 

```swift
// 3
let nc = customNavigationController([ .defaults, .showInMaster ])
nc.viewControllers = [viewController]

split.viewControllers = [nc]
```

has the correct expected behavior?

**Comment by nataliq** [comment]
Yes, exactly. See this: https://gist.github.com/nataliq/deef2efbd9ddab5e2b3bbf53dff9c0c0
I wanna file a radar but every time I go to the new feedback assistant, it's not working.

**Comment by nataliq** [comment]
I reported the issue through feedback assistant. Doesn't seem to be addressed in the latest betas too.

**Comment by nataliq** [comment]
Temporary fix implemented in 1.7.0 (https://github.com/iZettle/Presentation/pull/43)

**Comment by CJEkman** [comment]
Since it was discovered that this also affects `UINavigationController`, the dirty fix has been expanded to:
```Swift
shouldPresentImmediately = root is UISplitViewController || vc is UISplitViewController || (root as? UINavigationController)?.viewControllers.isEmpty == true
```
